### PR TITLE
Update badge for latest SDK version

### PR DESCRIPTION
**Pull request by OjesviPandita**
Update badge for latest SDK version

### CODE REVIEW

File: docs/clients.md [hunk 1]
```diff
@@ -222,9 +222,9 @@ Use this module for sending user generated actions to Ultra’s analytics librar
 #### Add the dependency:
 ***With Node or for React Native:***
 
-Add the [latest alpha version of SDK]((https://www.npmjs.com/package/fk-platform-sdk/v/1.0.3-analytics-sdk-alpha)) to your package.json file:
+Add the [latest version of SDK](https://www.npmjs.com/package/fk-platform-sdk/v/1.0.5) to your `package.json` file:
```
Comment (MSReehal) [inline docs/clients.md]: Latest version of SDK is 1.0.6 (and not 1.0.5). Please update.

File: docs/clients.md [hunk 2]
```diff
@@ -222,9 +222,9 @@ Use this module for sending user generated actions to Ultra’s analytics librar
...
 ```js
-"fk-platform-sdk": "1.0.3-analytics-sdk-alpha"
+"fk-platform-sdk": "1.0.5"
```
Comment (MSReehal) [inline docs/clients.md]: Latest version is 1.0.6.

File: docs/clients.md (hunk 1 above)
Comment (OjesviPandita) [reply docs/clients.md]: Done.

File: docs/clients.md (hunk 2 above)
Comment (OjesviPandita) [reply docs/clients.md]: Done.

**Pull request merged**